### DISCUSSION TITLE
PAT Basic 1035. 插入与归并 (25) (C语言实现) | PAT Solution in C

### DISCUSSION THREAD

**Discussion by xlucn** [xlucn/PAT#69] [General]
https://oliverlew.github.io/PAT/Basic/1035.html 

 题目

**Comment by yangfff** [comment]
大大您好，我看到算法笔记上指出这道题有一个陷阱：初始序列不参与是否与目标序列相同的比较（也就是说，题目中的 中间序列是不包括初始序列的）。不考虑这个陷阱会导致某个数据产生双解。大大您之前的代码没通过是不是和这个陷阱有关呢？
